packages/Heading/README.md

```diff
@@ -0,0 +1,68 @@
+## Heading
+
+The `<Heading>` component provides six different levels of heading with different stylings such as divider, underline, light or hidden.
+
+It is rendered as a `<h> tag` by default, but can also be rendered as a generic `<div>`. Set `isSemantic` to false if there are global CSS conflicts.
+
+### Installation
+
+`> npm install --save @paprika/heading`
+or
+`> yarn add @paprika/heading`
+
+### Usage
+
+For a basic heading
+
+```js
+import Heading from "@paprika/heading";
+
+<Heading level={1}>Heading One</Heading>;
+```
+
+Or a h6 but displayed as h1
+
+```js
+import Heading from "@paprika/heading";
+
+<Heading level={6} displayLevel={1}>
+  Heading Six
+</Heading>;
+```
+
+Or headings with divider, hasUnderline, isLight, or isHidden
+
+```js
+import Heading from "@paprika/heading";
+
+<Heading level={1} hasDivider>
+  Heading One With divider
+</Heading>;
+
+<Heading level={1} hasUnderline>
+  Heading One With underline
+</Heading>;
+
+<Heading level={1} isLight>
+  Heading One With isLight
+</Heading>;
+
+<Heading level={1} isHidden>
+  Heading One With isHidden
+</Heading>;
+```
```

Comment:
This is more detailed than our other README files.  I'm not saying we should remove it, but the README doesn't need to be comprehensive. This is a very simple component, but if we tried to match this level of detail for a more complex component, the README could be quite huge. There are also prop descriptions and Storybook to pick up where this leaves off, and ideally we wouldn't have too much redundancy, since it means more maintenance to keep everything up to date as we make changes.

Reply:
done